deployments.go

```diff
@@ -77,6 +77,34 @@ func (s *deployerService) Deploy(ctx context.Context, db *gorm.DB, opts DeployOp
 	if err := json.NewEncoder(opts.Output).Encode(&msg); err != nil {
 		return r, err
 	}
-
+	go s.streamEvents(opts)
```

Comment:
i initially had the blocking here, but was having issues with concurrent requests, so i moved it here: https://github.com/mhahn/empire/blob/c1ea348005d0c4eb832858810a87cf7926295118/server/heroku/deployments.go#L33. let me verify if that was the issue after i make some of these other changes.